resources/user-guides/app-extension.md

```diff
@@ -0,0 +1,74 @@
+# iOS App Extension Implementation
+
+You can use supported AEP extensions in your App Extensions to collect usage data. Supported Adobe Extensions are shown [here](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift). This tutorial assumes a basic understanding of how to use the iOS SDK in applications.
+
+## Adding the SDK
+
+Getting the Mobile SDK in your App Extension works the same way as for your application. For this tutorial we will go over Cocoapods as the method of installing the SDK.
```

Comment:
"Adding Mobile SDK to your App Extension works the same way as adding Mobile SDK to your application. This tutorial explains installing the SDK using Cocoapods."